src/main/scala/sbttravisci/TravisCiPlugin.scala

```diff
@@ -3,9 +3,26 @@ package sbttravisci
 import sbt._, Keys._
 
 object TravisCiPlugin extends AutoPlugin {
+  private val DefaultScalaVersion = "2.10.6"    // for compatibility with stock sbt
+
+  object autoImport {
+    val isTravisBuild = settingKey[Boolean]("Flag indicating whether the current build is running under Travis")
+  }
+
+  import autoImport._
+
   override def requires = plugins.JvmPlugin
   override def trigger  = allRequirements
   override def buildSettings = Seq(
+    isTravisBuild := sys.env.get("TRAVIS").isDefined,
```

Comment:
Env vars are global, so it would be even better to set this in globalSettings. Won't change anything for 99% of the cases, but it's more correct.l technically. Its in the docs: new key defaults should be set in globalSettings.

Also this is a great addition.